instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -236,7 +236,7 @@ def test_basic_metric_success(self):
                         dict(point.attributes),
                     )
                     self.assertEqual(point.count, 1)
-                    self.assertAlmostEqual(duration, point.sum, delta=20)
+                    self.assertAlmostEqual(duration, point.sum, delta=30)
```

Comment:
You could also use something like `@flaky(max_runs=..., min_sucess=...)` if this is small but eventually passes with a re-run.

Reply:
Hey @srikanthccv 
It sounds good as even `delta = 30` could be flaky. Should I create a PR and add this in all the frameworks where this kind of test is written??